src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1,6 +1,9 @@
-UPDATE tab_transport_directions SET display_name='Bi-directional plasma <-> blood cells', description='Bi-directional transport between blood plasma and blood cells' WHERE transport_direction='BiDirectionalBloodCellsPlasma';
-UPDATE tab_transport_directions SET display_name='Bi-directional interstitial <-> intracellular', description='Bi-directional transport between interstitial and intracellular space' WHERE transport_direction='BiDirectionalBrainInterstitialTissue';
-UPDATE tab_transport_directions SET display_name='Bi-directional plasma <-> interstitial', description='Bi-directional transport across the blood-brain-barrier' WHERE transport_direction='BiDirectionalBrainPlasmaInterstitial';
-UPDATE tab_transport_directions SET display_name='Bi-directional interstitial <-> intracellular', description='Bi-directional transport between interstitial and intracellular space' WHERE transport_direction='BiDirectionalInterstitialIntracellular';
-UPDATE tab_transport_directions SET display_name='Bi-directional lumen <-> mucosa', description='Bi-directional transport between intestinal lumen and mucosal cells' WHERE transport_direction='BiDirectionalLumenMucosaIntracellular';
-UPDATE tab_transport_directions SET display_name='Bi-directional plasma <-> interstitial', description='Bi-directional transport between blood plasma and interstitial space' WHERE transport_direction='BiDirectionalPlasmaInterstitial';
+UPDATE tab_calculation_method_rates SET formula='f_int > 0 ? RC * rel_exp_brn * f_exp_brn_tissue *  1 / f_int : 0' WHERE calculation_method='ExpressionParameters' AND formula_rate='InitialConcentrationBrainInterstitial_Transporter';
+UPDATE tab_calculation_method_rates SET formula='(f_vas * (1 - HCT)) > 0 ? RC * rel_exp_brn * f_exp_brn_bbb * 1 / (f_vas * (1 - HCT)) : 0' WHERE calculation_method='ExpressionParameters' AND formula_rate='InitialConcentrationBrainPlasma_Transporter';

```

Comment:
@Yuri05 I don't think I need the ( ) in `(f_vas * (1 - HCT))` but I figured it might be easier to read? 
Thoughts?

Reply:
agree (with better readability). indeed, () is not required here